src/server/server_family.cc

```diff
@@ -2064,6 +2134,30 @@ void ServerFamily::ReplicaOf(CmdArgList args, ConnectionContext* cntx) {
   }
 }
 
+void ServerFamily::ReplicaOf(CmdArgList args, ConnectionContext* cntx) {
+  string_view host = ArgS(args, 0);
+  string_view port = ArgS(args, 1);
+
+  // don't flush if input is NO ONE
+  if (!(absl::EqualsIgnoreCase(host, "NO") && absl::EqualsIgnoreCase(port, "ONE")))
```

Comment:
Maybe wrap this in a simple function, as it's now used in 2 places?
`IsReplicationOfNoOne()` or smtg

Reply:
sure!